Adafruit_AS7341.cpp

```diff
@@ -502,6 +502,31 @@ bool Adafruit_AS7341::setLEDCurrent(uint16_t led_current_ma) {
   return result;
 }
 
+/**
+ * @brief Get the current limit for the LED
+ *
+ * Range is 4mA to 258mA
+ * @return current limit in mA
+ */
+uint16_t Adafruit_AS7341::getLEDCurrent(void) {
+  uint16_t led_current_ma;
+  uint32_t led_raw;
+
+  setBank(true); // Access 0x60 0x74
+
+  Adafruit_BusIO_Register led_reg =
+      Adafruit_BusIO_Register(i2c_dev, AS7341_LED);
+
+  Adafruit_BusIO_RegisterBits led_current_bits =
+      Adafruit_BusIO_RegisterBits(&led_reg, 7, 0);
+
+  led_raw = led_current_bits.read();
+
+  led_current_ma = (uint16_t)(led_raw * 2) + 4;
```

Comment:
Could just return right hand side here instead of creating `led_current_ma` and returning that.

Reply:
Agreed. I tend to create "unnecessary" variables for the sake of being clear and documenting what the code does.